lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -4492,27 +4492,29 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
         if (activeValue->getType().isAddress()) {
           // Allocate and zero initialize a new local buffer using
           // `getAdjointBuffer`.
-          builder.setInsertionPoint(adjoint.getEntryBlock());
+          builder.setInsertionPoint(pullback.getEntryBlock());
           getAdjointBuffer(origBB, activeValue);
         } else {
-          // Create and register adjoint block argument for the active value.
-          auto *adjointArg = adjointBB->createPhiArgument(
+          // Create and register pullback block argument for the active value.
+          auto *adjointArg = pullbackBB->createPhiArgument(
               getRemappedTangentType(activeValue->getType()),
               ValueOwnershipKind::Guaranteed);
           activeValueAdjointBBArgumentMap[{origBB, activeValue}] = adjointArg;
         }
       }
       // Add a pullback struct argument.
-      auto *pbStructArg = adjointBB->createPhiArgument(
+      auto *pbStructArg = pullbackBB->createPhiArgument(
           pbStructLoweredType, ValueOwnershipKind::Guaranteed);
       pullbackStructArguments[origBB] = pbStructArg;
-      // - Create adjoint trampoline blocks for each successor block of the
+      // - Create pullback trampoline blocks for each successor block of the
       //   original block. Adjoint trampoline blocks only have a pullback
```

Comment:
Rename occurrences of `adjoint` to `pullback`.
Also fix last clause: `trampoline adjoint values` -> `passing adjoint values`.

```
      //   original block. Pullback trampoline blocks only have a pullback
      //   struct argument, and branch from the pullback successor block to the
      //   pullback original block, passing adjoint values of active values.
```